src/core/thread/key_manager.cpp

```diff
@@ -633,7 +633,18 @@ void KeyManager::SetNetworkKeyRef(otNetworkKeyRef aKeyRef)
 exit:
     return;
 }
+#endif // OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
+
+#if OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
+void KeyManager::DestroyTemporaryKeys(void)
```

Comment:
Combine the `#if` blocks?
```suggestion

void KeyManager::DestroyTemporaryKeys(void)
```